This reverts https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/11714,
which implements resource_create_front for vtest.

It breaks `glxinfo` with `virglrenderer-android`.

--- a/src/gallium/winsys/virgl/vtest/virgl_vtest_winsys.c
+++ b/src/gallium/winsys/virgl/vtest/virgl_vtest_winsys.c
@@ -369,7 +369,11 @@ virgl_vtest_winsys_resource_create(struct virgl_winsys *vws,
    }
 
    res->res_handle = new_handle;
+#ifndef __TERMUX__
    if (map_front_private && res->ptr && res->dt) {
+#else
+   if (map_front_private && res->ptr && res->dt && getenv("TERMUX_ENABLE_GLXCOPYSUBBUFFERMESA_FOR_VTEST")) {
+#endif
       void *dt_map = vtws->sws->displaytarget_map(vtws->sws, res->dt, PIPE_MAP_READ_WRITE);
       uint32_t shm_stride = util_format_get_stride(res->format, res->width);
       util_copy_rect(res->ptr, res->format, shm_stride, 0, 0,
